pom.xml

```diff
@@ -521,41 +521,8 @@
             <name>Spring Milestones</name>
             <url>https://repo.spring.io/milestone</url>
         </repository>
-        <repository>
-            <id>jfrog-flock-community</id>
-            <name>jfrog-repositories</name>
-            <url>https://flock.jfrog.io/artifactory/flock-maven</url>
-            <snapshots>
-                <enabled>false</enabled>
-            </snapshots>
-        </repository>
     </repositories>
 
-    <pluginRepositories>
-        <pluginRepository>
-            <snapshots>
-                <enabled>false</enabled>
-            </snapshots>
-            <id>jfrog-flock-community</id>
-            <name>jfrog-plugins</name>
-            <url>https://flock.jfrog.io/artifactory/flock-maven</url>
-        </pluginRepository>
-    </pluginRepositories>
-
-
-    <distributionManagement>
-        <repository>
-            <id>jfrog-flock-community</id>
-            <name>jfrog-repo-releases</name>
-            <url>https://flock.jfrog.io/artifactory/flock-maven</url>
-        </repository>
-        <snapshotRepository>
-            <id>jfrog-flock-community</id>
-            <name>jfrog-repo-snapshots</name>
-            <url>https://flock.jfrog.io/artifactory/flock-maven</url>
-        </snapshotRepository>
-    </distributionManagement>
-
```

Comment:
Boyscouting, removing all `jfrog` references as packages are published to maven central